sechub-doc/src/docs/asciidoc/documents/pds/product_delegation_server_config_example1.json

```
@@ -34,8 +37,15 @@
         {
             "id" : "PRODUCT_2",
             "path" : "/srv/security/scanner2.sh",
-            "scanType" : "infraScan" ,
-            "extractUploads" : false //<11>
+            "scanType" : "infraScan",
+            "parameters" : {
+                "mandatory" : [
+                    {
+                        "key" : "pds.config.supported.datatype", //<12>
```

Please see the comment above above:
```suggestion
                        "key" : "pds.config.supported.datatypes", //<12>
```